recipes/ledger/meta.yaml

```diff
@@ -0,0 +1,65 @@
+{% set name = "ledger" %}
+{% set version = "3.1.3" %}
+{% set sha256 = "b248c91d65c7a101b9d6226025f2b4bf3dabe94c0c49ab6d51ce84a22a39622b" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/ledger/ledger/archive/v{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+  patches:
+    - fix-build.patch
+
+build:
+  number: 0
+  skip: True  # [win or osx or py>27]
```

Comment:
Add,
```yaml
  run_exports:
    - {{ pin_subpackage("ledger") }}
```

Reply:
Could you also explain this suggestion a little further?

Reply:
When a downstream library links to this library, this will add a version constraint for `ledger` in the downstream library so that the major version is pinned. This is to ensure that downstream packages linking to `libledger.so.3` will always use the major version 3